FileProcessingSystem/src/com/group7/fileprocessingsystem/FileProcessingSystem.java

```diff
@@ -1,18 +1,19 @@
-/*
- * ENGG*1420 - Group 7
- * Feb 28, 2023
- * The main class for the File Processing System
- */
-package com.group7.fileprocessingsystem;
-
-/**
- *
- * @author Tacitor
- */
-public class FileProcessingSystem {
-    
-    public static void main(String[] args) {
-        System.out.println("Stating file processor...");
-    }
-    
-}
+/*
+ * ENGG*1420 - Group 7
+ * Feb 28, 2023
+ * The main class for the File Processing System
+ */
+package com.group7.fileprocessingsystem;
+
+/**
+ *
+ * @author Tacitor
+ */
+public class FileProcessingSystem {
+    
+    public static void main(String[] args) {
+        System.out.println("Stating file processor...");
+        System.out.println("Eric's Branch");
+    }
+    
+}
```

Comment:
Should delete this entire project and just used the Maven one

Reply:
done. see new commit.